app/models/order.rb

```diff
@@ -0,0 +1,15 @@
+class Order < ApplicationRecord
+    belongs_to :user
+    has_many :order_items
+    before_save :set_subtotal
+    
+    def subtotal
```

Comment:
Do not override rails provided methods. calculate subtotal from set_subtotal

Reply:
Okay i will take care of it.